### PR TITLE
Merge 11.3.0 into 12.0.0

### PR DESCRIPTION
**Pull request by Klaas-Ritense**
DONT SQUASH

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -203,18 +206,30 @@
       * [Migration](release-notes/major12/12.0.0/migration.md)
         * [Spring Boot 3](release-notes/major12/12.0.0/spring-boot3-migration.md)
   * 11.x.x
+    * 11.3.0
+      * [Valtimo backend libraries](release-notes/major11/11.3.0/valtimo-backend-libraries.md)
+      * [Valtimo frontend libraries](release-notes/major11/11.3.0/valtimo-frontend-libraries.md)
     * 11.2.0 (RC)
       * [Valtimo backend libraries](release-notes/major11/11.2.0/valtimo-backend-libraries.md)
       * [Valtimo frontend libraries](release-notes/major11/11.2.0/valtimo-frontend-libraries.md)
-      * [Migration](release-notes/major11/11.2.0/migration.md)
```
Comment (theo-ritense) [inline SUMMARY.md:209]: Was this intended to be removed?

File: extending-valtimo/form-flow/form-flow.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Form flow
+# Form flow (deprecated)
```
Comment (theo-ritense) [inline extending-valtimo/form-flow/form-flow.md:1]: deprecated? Did I miss the memo?

Comment (Klaas-Ritense) [reply extending-valtimo/form-flow/form-flow.md:1]: no, definitely not

From what I can see, this mistake was made in `development/11.2.0` and its now also wrong in `development/11.3.0`. It has to be fixed there so we can merge it back later

Comment (theo-ritense) [reply extending-valtimo/form-flow/form-flow.md:1]: Ok. Merge this one and I'll see if I can fix that retroactively.

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**